com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -479,19 +479,13 @@ else if (severity == MultipleCompileException.Info.SEVERITY_WARNING)
         }
     }
 
-    public static void createClassLoaderScanner() throws IOException {
-        // Find the jar file in the built-in resources
-        String jar = Bob.getJarFile("fmt-spine.jar");
-        scanner = new ClassLoaderScanner();
-        scanner.addUrl(new File(jar));
-    }
-
     public static ClassLoaderScanner getClassLoaderScanner() {
         return scanner;
     }
 
     private static void setupProject(Project project, boolean resolveLibraries, String sourceDirectory) throws IOException, LibraryException, CompileExceptionError {
-        createClassLoaderScanner();
+        scanner = Project.createClassLoaderScanner();
```

Comment:
Note: The editor doesn't really invoke "bob", but a manually created instance of the Project class.